core/shared/src/main/scala/zio/Exit.scala

```diff
@@ -206,6 +206,12 @@ object Exit extends Serializable {
         .map(_.reverse)
     }
 
+  /**
+   *  Alias for `[[ZIO.collectAll]]`.
```

Comment:
Thanks! the backticks are unnecessary when you have the square brackets

Reply:
I generated the docs and checked the links work

Reply:
Ah that's great then